README.md

```diff
@@ -74,9 +74,11 @@ github "CombineCommunity/CombineExt"
 
 This section outlines some of the custom operators CombineExt provides.
 
-### withLatestFrom
+## withLatestFrom
 
-Merges two publishers into a single publisher by combining each value from `self` with the _latest_ value from the second publisher, if any.
+### withLatestFrom(_:)
+
+Merges **two** publishers into a single publisher by combining each value from `self` with the _latest_ value from the second publisher, if any.
```

Comment:
Let's not add specific documentation per-overload:) 
We should just expand withLatestFrom mentioning it supports overloads of up to 4 observables

Reply:
I updated te README based on this comment and I hope I got it right. Should it be fine now? 😄